scripts/qa_crowbarsetup.sh

```diff
@@ -1350,6 +1350,24 @@ function custom_configuration()
             if [ -n "$networkingplugin" ] ; then
                 proposal_set_value neutron default "['attributes']['neutron']['networking_plugin']" "'$networkingplugin'"
             fi
```

Comment:
> I think you need to put the 6 lines above in a else.

Done in https://github.com/toabctl/automation/commit/69a064c3fe4e5cc592a7f4e6e4dcc9e42d22ef9a